resources/translations/wb_it.ts

```diff
@@ -1032,101 +1033,103 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
     <message>
         <source>Move Viewpoint to see object from the left.</source>
-        <translation type="unfinished"></translation>
+        <translation>Muovi il Punto di Vista per vedere l&apos;oggetto da sinistra.</translation>
     </message>
     <message>
         <source>&amp;Right View</source>
-        <translation type="unfinished"></translation>
+        <translation>Vista &amp;Destra</translation>
     </message>
     <message>
         <source>Move Viewpoint to see object from the right.</source>
-        <translation type="unfinished"></translation>
+        <translation>Muovi il Punto di Vista per vedere l&apos;oggetto da destra.</translation>
     </message>
     <message>
         <source>&amp;Top View</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Vista Sopra</translation>
     </message>
     <message>
         <source>Move Viewpoint to see object from the top.</source>
-        <translation type="unfinished"></translation>
+        <translation>Muovi il Punto di Vista per vedere l&apos;oggetto dal sopra.</translation>
     </message>
     <message>
         <source>&amp;Bottom View</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Vista Sotto</translation>
     </message>
     <message>
         <source>Move Viewpoint to see object from the bottom.</source>
-        <translation type="unfinished"></translation>
+        <translation>Muovi il Punto di Vista per vedere l&apos;oggetto dal sotto.</translation>
     </message>
     <message>
         <source>&amp;Take Screenshot...</source>
-        <translation type="unfinished">Cattura Schermata...</translation>
+        <translation>&amp;Cattura Schermata...</translation>
     </message>
     <message>
         <source>Save the current image of the simulation. (%1 + SHIFT + P)</source>
-        <translation type="unfinished"></translation>
+        <translation>Salva l&apos;immagine corrente della simulazione. (%1 + SHIFT + P)</translation>
     </message>
     <message>
         <source>Move viewpoint to selected object. (ALT + 5)</source>
-        <translation type="unfinished"></translation>
+        <translation>Muovi il punto di vista verso l&apos;oggetto selezionato. (ALT + 5)</translation>
     </message>
     <message>
         <source>&amp;Reload World</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Ricarica Scenario</translation>
     </message>
     <message>
         <source>Reload World.
 Reload the current world file and restart the simulation. (%1+Shift+R)</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricarica Scenario.
+Ricarica lo scenario corrente e ricomincia la simulazione. (%1+Shift+R)</translation>
     </message>
     <message>
         <source>Reset Simulation</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione</translation>
```

Comment:
I would rather use "azzera" than "resetta" (that is more a slang)
```suggestion
        <translation>Azzera Simulazione</translation>
```